Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Helpers/ColourHelper.cs

```diff
@@ -28,104 +28,104 @@ public static class ColourHelper
         /// <summary>
         /// HSV Structure
         /// </summary>
-        public struct HSVColour
+        public struct HsvColour
         {
-            private float hue;
+            private float _hue;
 
             /// <summary>
             /// Hue value, from 0° to 360°
             /// </summary>
             public float Hue
             {
-                get => hue;
-                set => hue = value;
+                get => _hue;
+                set => _hue = value;
             }
 
-            private float saturation;
+            private float _saturation;
 
             /// <summary>
             /// Saturation value, from 0 to 1
             /// </summary>
             public float Saturation
             {
-                get => saturation;
-                set => saturation = value;
+                get => _saturation;
+                set => _saturation = value;
             }
 
-            private float valueOrBrightness;
+            private float _valueOrBrightness;
 
             /// <summary>
             /// Value or Brightness value, from 0 to 1
             /// </summary>
             public float ValueOrBrightness
             {
-                get => valueOrBrightness;
-                set => valueOrBrightness = value;
+                get => _valueOrBrightness;
+                set => _valueOrBrightness = value;
             }
 
             /// <summary>
-            /// Initializes a new instance of the <see cref="HSVColour"/> struct.
+            /// Initializes a new instance of the <see cref="HsvColour"/> struct.
             /// </summary>
             /// <param name="hue">The hue.</param>
             /// <param name="saturation">The saturation.</param>
             /// <param name="value">The value.</param>
-            public HSVColour(float hue, float saturation, float value)
+            public HsvColour(float hue, float saturation, float value)
             {
-                this.hue = hue;
-                this.saturation = saturation;
-                this.valueOrBrightness = value;
+                _hue = hue;
+                _saturation = saturation;
+                _valueOrBrightness = value;
             }
         }
 
         /// <summary>
         /// HSL Structure
         /// </summary>
-        public struct HSLColour
+        public struct HslColour
```

Comment:
`HSL` needs to be a valid string naming type

Reply:
Done